dev/devicelab/lib/tasks/hot_mode_tests.dart

```diff
@@ -257,3 +269,19 @@ Future<Map<String, dynamic>> captureReloadData(
   benchmarkFile.deleteSync();
   return result;
 }
+
+Future<void> _checkAppRunning(bool shouldBeRunning) async {
```

Comment:
Something tells me this will be a flake factory...  Will delete if that starts happening.

Reply:
agreed